src/components/Button/index.tsx

```diff
@@ -80,6 +82,7 @@ export const Button = ({
                 ButtonRounded[rounded],
                 {
                     "deriv-button__full-width": isFullWidth,
+                    "deriv-button__disable-hover": !hasHoverStyle,
```

Comment:
following my above message: 
```suggestion
                    "deriv-button__disable-hover": hideHoverStyles,
```